ui/docs/how_to_use_the_ui.md

```diff
@@ -0,0 +1,150 @@
+   - **Purpose:** Define the number and type of GPUs or accelerators to use.  
+   - **Configuration:** Use the slider to select the number of accelerators (range: 08).
+
+2. **Number of Nodes:**  
+   - **Purpose:** Specify the number of compute nodes.  
+   - **Configuration:** Adjust the slider (range: 14).
+
+3. **Accelerator Type:**  
+   - **Purpose:** Select the type of accelerator.  
+   - **Configuration:** Use the multi-select dropdown for supported models, such as NVIDIA H100.
+
+4. **Host Processor Core Count:**  
+   - **Purpose:** Specify the number of CPU cores available on the host system.  
+   - **Configuration:** Adjust the slider (range: 0128).
+
+5. **Model MLC:**  
+   - **Purpose:** Select the AI model being evaluated.  
+   - **Configuration:** Use the dropdown to choose a model, e.g., `llama2-70b`.
+
+6. **Processor Type:**  
+   - **Purpose:** Define the processor type for computations.  
+   - **Configuration:** Choose from available options, such as Intel Xeon or AMD EPYC.
+
+7. **Scenario:**  
+   - **Purpose:** Define the workload scenario.  
+   - **Configuration:** Select options like *Offline* or *Server*.
+
+---
+
+## Configuring Output Parameters
+
+The **Output Configuration Panel** allows users to select prediction metrics. For the `MLCommons` dataset, available outputs include:
+
+1. **Tokens Per Second:**  
+   - **Purpose:** Display the model's token generation rate based on the configuration.  
+   - **Configuration:** Check the box next to this metric.
+
+> **Note:** Multiple outputs can be selected for a comprehensive analysis.
+
+---
+
+## Interpreting the Results Table
+
+After configuring inputs and outputs, click the **Predict** button. Results appear in the **Main Display Area**. For the `MLCommons` dataset, the table includes:
+
+- **Resource Estimation:** Expected hardware and node usage.
+- **Performance Metrics:** Metrics like tokens per second.
+- **Scenario Details:** A summary of input configurations.
+
+### Example Table:
+
+| Metric               | Value         |
+|-----------------------|---------------|
+| Tokens Per Second     | 4500          |
+| Accelerators Used     | 3             |
+| Accelerator           | NVIDIA H100   |
+| CPU Cores             | 42            |
+
+---
+
+## Using Filters for Analysis
+
+The result table includes filtering features to refine results:
+
+- **Sort by Metric:**  
+  Organize data based on performance or resource usage, either ascending or descending.
+
+- **Dropdown Filters:**  
+  Refine results based on categories (e.g., accelerator type) or numerical ranges (e.g., tokens per second).  
+
+### How to Use:
+1. Click the **Add Filters** button to open a filter selection combo box.
+2. Select the column to filter.
+3. Use the dropdown or slider to narrow the data focus.
+
+---
+
+## Example Use Case
+
+**Scenario:** Estimate the resources required to achieve 5000 tokens per second for an AI model using NVIDIA H100 GPUs.
```

Comment:
Add a comment that a more elaborate example and explanation of what happens under the hood can be found at ../doc/example.md

Reply:
fixed